frontend/packages/operator-lifecycle-manager/src/components/operand/DEPRECATED_operand-form.tsx

```diff
@@ -553,7 +553,7 @@ export const DEPRECATED_CreateOperandForm: React.FC<OperandFormProps> = ({
         // Add the field if it doesn't exist
         return [
           ...providedAPIFieldsAccumulator,
-          ...specDescriptorToFields(specDescriptor, formData),
+          ...specDescriptorToFields(specDescriptor, formData as K8sResourceKind),
```

Comment:
Don't cast to K8sResourKind, figure out what's wrong with the underlying type definition.